tf2_eigen/include/tf2_eigen/tf2_eigen.h

```diff
@@ -34,19 +34,33 @@
 #include <Eigen/Geometry>
 #include <geometry_msgs/msg/point_stamped.hpp>
 #include <geometry_msgs/msg/pose_stamped.hpp>
+#include <geometry_msgs/msg/quaternion_stamped.hpp>
```

Comment:
Aren't these pulled in by the previous headers?

Reply:
Yes, but if we use them in this file we should include them here too.